src/main/java/trackitnus/ui/MainWindow.java

```diff
@@ -182,7 +182,7 @@ private void switchTab(ArrayList<Object> tabValues) throws CommandException {
             tabPanelPlaceholder.getChildren().add(modulePanel.getRoot());
             break;
         case Contact.TYPE:
-            contactPanel = new ContactPanel(logic);
+            contactPanel = new ContactPanel(logic.getAllContacts());
```

Comment:
Can I check how come suddenly need to add this?

Reply:
just for better code quality passing in just the observable list to show instead of the entire logic instance